test/runtests.jl

```diff
@@ -1704,3 +1704,10 @@ end
     @test length(StaticLint.loose_refs(bindingof(cst[1][3][1][3][1][1]))) == 2
     @test length(StaticLint.loose_refs(bindingof(cst[1][3][3][1]))) == 2
 end
+
+@testset "#1218" begin 
+    cst = parse_and_pass("""function foo(a; p) end
+    foo(1, p = true)""")
+    @test isempty(StaticLint.collect_hints(cst, server))
```

Comment:
Actually, this should throw "unused argument" warnings, no?

Reply:
Ah yes, that's odd that it isn't - will have a look